_includes/sponsor-template.html

```diff
@@ -11,41 +11,46 @@ <h1 class="gala-heading">{{sponsor_type.sponsor-heading}}</h1>
     <!-- sponsors by level, such as 白金赞助 -->
     {% for sponsor_level in sponsor_type.sponsor-levels %}
     <div class="row">
-        <div class="col home-section-header pb-small">
```

Comment:
Don't hard-code CSS into HTML. Also, can you use a `pt-4` (or `pt-6`; choose the appropriate number) class that is already exist in Bootstrap instead of writing `padding-top` manually?